### ISSUE TITLE
your library out of update

### ISSUE THREAD

**Issue by hafiz013** [begeekmyfriend/yasea#660]
hi i reference your library out of update which is library c++ such as mips64el-linux-android-4.9 not more supported in coming version cmake. I also update your source code to the latest api 7.1 android above. Can i push my code to your server.

**Comment by begeekmyfriend** [comment]
Any improved PR is welcome!

**Comment by hafiz013** [comment]
how to push my code to your server?

**Comment by begeekmyfriend** [comment]
Fork my repo and add your modification on the master branch, and then create the PR on Github.

**Comment by hafiz013** [comment]
already update your library. Please remove this folder zip 'android-eye-master-update.zip'

**Comment by hafiz013** [comment]
nevermind I already delete that folder. Please update your document which are required to import c++ mips64el-linux-android-4.9 into latest ndk bundle. This file can get from version 1.6 and 1.7

**Comment by begeekmyfriend** [comment]
Pardon that where should I update in my repo?

**Comment by hafiz013** [comment]
https://github.com/hafiz013/yasea . <-- this repo i update. I already push my code but i dont know how to update the your code.

**Comment by begeekmyfriend** [comment]
Just click `New pull request` button and I will receive a PR notification. Then I can see what commits you have updated

**Comment by hafiz013** [comment]
never mind, please download whole this file : https://drive.google.com/file/d/10BcfEKLNoNjbt89LXymlPD1GDU3VKofl/view?usp=sharing.

because i pull and request and push still goes to my personal repo.

Note: Update library folder, mainactivity (added run time permission), gradle change from compile to implementation. That's all please update. Thank you.. However, I still got blur how to reduce noise sound when doing live.

**Comment by begeekmyfriend** [comment]
Sorry that I did not find any modification in your zip file on C++ part of library. Did you mean the modification is only in `build.gradle`?

**Comment by hafiz013** [comment]
yup correct modification in build.gradle for library folder and apps folder. I also put run time permission in main activity in order to allow run camera and record sound. For c++, I just mention that mips64el-linux-android-4.9 no more support in latest android studio. So you need to update your documentation to tell user to have download this folder manually and added to bundle-ndk manually for latest android studio.
